### ISSUE TITLE
Interference with ReMemorize

### ISSUE THREAD

**Issue by galantra** [hssm/advanced-browser#82]
Hi,

an error occurred when I tried to reschedule a card from the reviewer by means of the two add-ons ReMemorize and ReMemorize buttons.

```
Error
An error occurred. Please start Anki while holding down the shift key, which will temporarily disable the add-ons you have installed.
If the issue only occurs when add-ons are enabled, please use the Tools > Add-ons menu item to disable some add-ons and restart Anki, repeating until you discover the add-on that is causing the problem.
When you've discovered the add-on that is causing the problem, please report the issue on the add-on support site.
Debug info:
Anki 2.1.22 (0ecc189a) Python 3.8.0 Qt 5.14.1 PyQt 5.14.1
Platform: Windows 10
Flags: frz=True ao=True sv=1
Add-ons, last update check: 2020-04-14 22:09:34

Caught exception:
Traceback (most recent call last):
  File "C:\Users\user\PortableApps\AnkiPortable\Anki2\addons21\323586997\rememorize.py", line 79, in _forgetCards
    self._finished(card,"%d Card(s) forgotten."%cnt)
  File "C:\Users\user\PortableApps\AnkiPortable\Anki2\addons21\323586997\rememorize.py", line 204, in _finished
    mw.reset()
  File "aqt\main.py", line 633, in reset
  File "aqt\gui_hooks.py", line 1560, in __call__
  File "aqt\browser.py", line 823, in onReset
AttributeError: 'AdvancedBrowser' object has no attribute 'editor'

```

Debug info:
    ```

    Anki 2.1.22 (0ecc189a) Python 3.8.0 Qt 5.14.1 PyQt 5.14.1
    Platform: Windows 10
    Flags: frz=True ao=True sv=1
    Add-ons, last update check: 2020-04-14 22:09:34
    
    
    ===Add-ons (active)===
    (add-on provided name [Add-on folder, installed at, version, is config changed])
    '' ['1135507716 tagedit_subtag_completer', 0, 'None', '']
    '' ['autoLearningSteps', 0, 'None', '']
    '' ['rememorize_btn', 0, 'None', mod]
    '' ['show_warning_if_no_tags_when_adding_cards', 0, 'None', '']
    '' ['touch_wood', 0, 'None', '']
    3 add-ons merged quicker anki explain deletion explain database check ['777545149', 2019-09-28T08:05, 'None', '']
    3ft Under Bury Newly Added Cards ['32017685', 2019-08-19T21:16, 'None', mod]
    Add Hyperlink ['318752047', 2020-04-03T18:06, 'None', '']
    Add Parent Tag To Every Tag In A Deck ['543106186', 2019-11-21T13:20, 'None', '']
    Add Table ['1237621971', 2019-11-08T21:17, 'None', mod]
    Add note id ['1672832404', 2017-11-15T11:32, 'None', '']
    Advanced Copy Fields ['1898445115', 2019-08-17T11:52, 'None', '']
    Agent Orange Pseudo-Fuzz Defoliator ['175851166', 2019-08-13T09:25, 'None', '']
    Anki Killstreaks ['579111794', 2020-03-10T18:20, 'None', '']
    Anki Zoom ['538879081', 2020-04-04T02:20, 'v1.1.3', mod]
    AnkiConnect ['2055492159', 2020-04-10T06:16, 'None', '']
    Answer Button Sounds ['679615590', 2020-02-20T16:10, 'None', '']
    Batch Editing ['291119185', 2019-08-08T19:56, 'None', '']
    Blitzkrieg II - Advanced Browser Sidebar BETA7 ['564851917', 2020-02-26T08:52, 'None', '']
    Browser TableEditor side-by-side horizontal split ['831846358', 2020-01-14T00:07, 'None', mod]
    Bury related notes ['413416269', 2020-04-03T09:55, 'None', mod]
    Butler-Addon ['1439937507', 2020-03-25T11:28, 'None', '']
    Cacher In The Rye Windows ['1302452246', 2020-01-16T18:48, 'None', '']
    Cardistry Dynamically Adjust New Cards ['1535078906', 2020-02-16T10:48, 'None', '']
    Change note type respect field and card names ['513858554', 2020-03-09T09:47, 'None', '']
    Change subdecks configuration ['751420631', 2019-11-17T08:01, 'None', '']
    Close and lose current input  for sticky fields ['1284221829', 2019-10-16T09:07, 'None', '']
    Cloze Overlapper ['cloze_overlapper', 2019-02-01T13:37, 'None', '']
    Copy notes ['1566928056', 2020-03-12T01:56, 'None', mod]
    Correct Answers Graph ['993120073', 2020-03-24T13:57, 'None', '']
    Correcting a bug in anki which makes new card appearing in wrong order ['127334978', 2019-12-09T08:43, 'None', '']
    Create link in note to display specific search  in browser ['2053932397', 2020-03-29T19:56, 'None', '']
    CrowdAnki JSON exportimport Edit history Collaborate on deck creation ['1788670778', 2020-02-10T16:24, 'None', '']
    Customize Keyboard Shortcuts ['24411424', 2020-03-11T06:05, 'None', mod]
    Dancing Baloney Allows changing wallpapers with live preview ['378638814', 2020-03-27T06:28, 'None', mod]
    Deck name in title 21 ['699175524', 2019-08-23T23:06, 'None', '']
    Delete empty NEW cards tag other empty cards ['1402327111', 2020-03-15T14:12, 'None', '']
    Do not underline current queue in reviewer ['267531099', 2019-08-17T12:23, 'None', '']
    Dont remove mark on export ['909480379', 2020-02-16T10:53, 'None', '']
    Ease Factor Histogram ['494944895', 2019-08-07T22:57, 'None', '']
    Empty cards returns more usable informations ['25425599', 2019-12-07T17:03, 'None', '']
    Enhance main window ['877182321', 2020-02-16T10:48, 'None', mod]
    Ensure each note is unique correct a bug introduced by add-on copy note ['2082040683', 2019-09-28T08:09, 'None', '']
    Extended Card Stats During Review ['1008566916', 2019-08-15T10:43, 'None', '']
    Extended Schema Hash ['1187737890', 2020-01-06T18:25, 'None', '']
    Extended Tag AddEdit Dialog ['1135507717', 2020-03-25T23:31, 'None', '']
    External Note Editor for the Browser ['1996266159', 2020-03-31T22:59, 'None', '']
    Forked Improved Quzlet to Anki Importer english ['1302146328', 2019-11-24T07:56, 'None', '']
    Frozen Fields ['516643804', 2020-03-01T05:12, 'None', '']
    Graphical form to change add-on configuration ['1014777615', 2020-03-19T11:22, 'None', '']
    HUMBLE PIE distinct cards studied today statistic ['2133933791', 2020-03-24T19:05, 'None', '']
    High Yield Tags ['1918380616', 2020-03-11T00:11, 'None', mod]
    Highlight Search Results in the Browser ['225180905', 2019-08-15T18:52, 'None', '']
    Hitmarkers ['1776869150', 2020-04-01T00:04, 'None', '']
    Hoarders Delight RecycleTrash Bin ['1758624338', 2019-08-13T10:59, 'None', '']
    Holdem Cardfield Preserves add card history ['363320830', 2019-08-13T10:23, 'None', '']
    HoochieBaby Queue Controller  Day-Lrn Q ['1847358755', 2020-04-14T05:35, 'None', '']
    HoochieMama Randomize Rev Queue ['1460733408', 2020-04-14T05:37, 'None', '']
    HoochiePapa Randomize New Queue ['1173108619', 2020-04-14T05:38, 'None', '']
    If a note has no more card warns instead of deleting it ['2018640062', 2019-08-13T10:19, 'None', '']
    Ignore accents in browser search ['1924690148', 2019-08-07T22:32, 'None', '']
    Image Occlusion Enhanced for Anki 21 alpha ['1374772155', 2019-09-11T14:53, 'None', '']
    ImageResizer ['1214357311', 2019-08-07T23:43, 'None', '']
    Improve speed of change of note type ['115825506', 2019-11-17T08:01, 'None', '']
    Improved Quizlet to Anki 21 Importer ['538351043', 2020-04-14T20:11, 'None', '']
    Increase autosave frequency ['786971082', 2019-08-13T18:26, 'None', mod]
    Keep card displayed in reviewer until it is modified ['1894745652', 2020-01-25T09:49, 'None', '']
    Keep files git svn in add-on folders ['225953877', 2020-03-09T09:55, 'None', '']
    Leaderboard ['41708974', 2020-04-14T09:27, 'None', mod]
    Load Balanced Scheduler ['208879074', 2019-08-08T20:32, 'None', '']
    Long term backups ['529955533', 2019-08-08T20:16, 'None', '']
    Markdown Support - With Batch n Preview ['1259478414', 2020-03-18T01:11, 'None', mod]
    Mini Format Pack ['295889520', 2019-08-07T21:49, 'None', mod]
    Minimal Flat Big Buttons compatible with ReMemorize Buttons ['1042429613', 2020-03-11T12:09, 'None', '']
    More Decks Stats and Time Left ['1556734708', 2020-03-31T22:19, 'None', '']
    No Distractions Full Screen Clean review interface  TabletTouch support ['1049863218', 2020-03-31T18:33, 'v4.1.1', mod]
    OpenInExternalEditorRenameDuplicate for ImageAudioVideo ['1560623518', 2020-04-05T11:25, 'None', mod]
    Opening the same window multiple time ['354407385', 2020-03-09T12:57, 'None', '']
    Partial synchronization for inconsistent database ['1675325364', 2020-03-06T06:33, 'None', '']
    Polar Connect - PDF and Incremental Reading with Anki Sync ['734898866', 2019-08-07T22:36, 'None', '']
    Preview any cloze number ['915063177', 2020-02-28T14:01, 'None', '']
    Progress Graphs and Stats for Learned and Matured Cards ['266436365', 2020-03-29T08:26, 'None', '']
    Puppy Reinforcement ['1722658993', 2020-03-03T08:49, 'None', '']
    Push Paradox Sibling Cardistry ['1797395052', 2020-04-04T06:40, 'None', '']
    ReMemorize Ex Rated0Search ['1056004913', 2019-08-09T09:34, 'None', '']
    ReMemorize Rescheduler with sibling and logging v132 ['323586997', 2020-04-04T07:11, 'None', mod]
    Regex Search  Wildcard Matching on Field Names ['2044559350', 2020-02-09T18:49, 'None', '']
    Reloadbackup menu option ['883617253', 2020-02-10T19:06, 'None', '']
    Removes Empty Note Types for 21 ['530953383', 2019-08-17T08:30, 'None', '']
    Reviewer to Browser choosing what to display ['1555020859', 2020-03-10T22:48, 'None', '']
    Search and Replace Tags ['138501288', 2019-09-12T10:23, 'None', '']
    Searching PDF Reading  Note-Taking in Add Dialog ['1781298089', 2020-04-01T23:36, 'None', mod]
    Slackers Delight PostponeDefer Button ['880824023', 2020-02-27T20:59, 'None', '']
    Statistics for Note Types Card Types and Tags ['16703369', 2019-12-17T09:22, 'None', '']
    Straight Reward ['957961234', 2020-03-17T18:38, 'None', mod]
    SyncRedirector ['2124817646', 2020-04-07T08:10, 'None', '']
    Tag Entry Enhancements ['1348430474', 2019-09-13T15:32, 'None', '']
    Text Rendering Fix ['94394764', 2018-11-01T13:48, 'None', '']
    True Retention by Card Maturity ['923360400', 2019-08-10T10:00, 'None', '']
    add-on dialog searchfilter bar ['561945101', 2020-01-21T23:42, 'None', '']
    avgEase ['633293039', 2019-08-07T23:01, 'None', '']
    bury future new sibs ['930365039', 2019-11-25T09:52, 'None', mod]
    copy note contents to new Add window from Browser Reviewer Add EditCurrent ['759018284', 2019-12-09T08:43, 'None', mod]
    editor apply font color background color custom class custom style ['1899278645', 2020-01-23T09:14, 'None', '']
    extended table editor ['805891399', 2020-03-05T19:24, 'None', mod]
    extended template editor with save and diff also for html source of fields ['1043915942', 2019-11-25T12:13, 'None', mod]
    remove linebreaks  do word wrap fork for 21 ['2028179508', 2019-08-18T09:48, 'None', mod]
    
    ===IDs of active AnkiWeb add-ons===
    1008566916 1014777615 1042429613 1043915942 1049863218 1056004913 1135507717 115825506 1173108619 1187737890 1214357311 1237621971 1259478414 127334978 1284221829 1302146328 1302452246 1348430474 1374772155 138501288 1402327111 1439937507 1460733408 1535078906 1555020859 1556734708 1560623518 1566928056 16703369 1672832404 1675325364 1722658993 175851166 1758624338 1776869150 1781298089 1788670778 1797395052 1847358755 1894745652 1898445115 1899278645 1918380616 1924690148 1996266159 2018640062 2028179508 2044559350 2053932397 2055492159 2082040683 208879074 2124817646 2133933791 225180905 225953877 24411424 25425599 266436365 267531099 291119185 295889520 318752047 32017685 323586997 354407385 363320830 378638814 413416269 41708974 494944895 513858554 516643804 529955533 530953383 538351043 538879081 543106186 561945101 564851917 579111794 633293039 679615590 699175524 734898866 751420631 759018284 777545149 786971082 805891399 831846358 877182321 880824023 883617253 909480379 915063177 923360400 930365039 94394764 957961234 993120073
    
    ===Add-ons (inactive)===
    (add-on provided name [Add-on folder, installed at, version, is config changed])
      Advanced Review Bottom Bar   ['1136455830', 2020-04-15T05:01, 'None', '']
    '' ['1135507716 bettertags', 0, 'None', mod]
    '' ['No Penalties or Boosting', 0, 'None', '']
    '' ['PassFail (EN)', 0, 'None', '']
    '' ['ResetEZ', 0, 'None', '']
    '' ['more_answer_buttons', 0, 'None', mod]
    '' ['no_same_day_spacing_for_siblings', 0, 'None', '']
    '' ['pez_dispenser', 0, 'None', '']
    '' ['roommate_switch', 0, 'None', '']
    '' ['visual_feedback', 0, 'None', '']
    '' ['wordsworth', 0, 'None', '']
    A half-automatic Forvo Downloader ['1407979740', 2019-08-08T07:05, 'None', mod]
    Add Audio and Pronunciation from Merriam-Webster ['1878363231', 2019-10-25T17:15, 'None', '']
    Add a tag to notes with missing media ['2027876532', 2019-08-08T20:58, 'None', '']
    Adding note and changing note type become quicker ['802285486', 2019-12-07T17:03, 'None', '']
    Additional Card Fields Fork for 21 ['744725736', 2020-03-30T19:43, 'None', '']
    AddonBackup ['162231225', 2020-02-10T19:06, 'None', '']
    Advanced Browser ['874215009', 2020-02-21T10:49, 'None', mod]
    Advanced Card Styles ['1625103157', 2019-09-05T14:20, 'None', '']
    Allows empty first field during adding and import ['46741504', 2019-08-08T19:56, 'None', '']
    AltGr keyboard shortcut workaround ['1786482670', 2019-11-14T07:59, 'None', '']
    Alternative Dialog for ReMemorize Touch-Friendly ['2107899486', 2019-10-20T19:58, 'None', '']
    Always On Top ['1760080335', 2019-08-17T11:51, 'None', '']
    Amboss add on new ['68362429', 2019-10-27T12:29, 'None', '']
    Anki Excel Sync ['1832015432', 2020-01-03T08:01, 'None', '']
    Anki Fanfare GamificationFeedback ['2073603704', 2020-02-07T09:31, 'None', mod]
    Anki Habitica for 21 ['1758045507', 2020-03-24T09:28, 'None', '']
    Anki Welcome Screen ['314871488', 2019-11-25T09:50, 'None', mod]
    Another Retreat go upstream the history for the next interval at lapse ['1481634779', 2019-08-07T22:57, 'None', '']
    Audio Files ['1984823157', 2019-12-30T17:44, 'None', mod]
    Auto Markdown ['1030875226', 2019-11-15T19:18, 'None', mod]
    AutoDefine - Automatically define vocabulary words with pronunciations and image ['2136497008', 2019-11-14T15:23, 'None', mod]
    Automatic Basic to Cloze ['2105427255', 2019-09-17T16:20, 'None', '']
    AwesomeTTS Google Cloud Text-to-Speech unofficial ['814349176', 2020-04-14T14:51, 'None', '']
    Batch Download Pictures From Google Images ['561924305', 2020-04-08T11:30, 'None', '']
    Bluetooth Headset Audio Lag Fix ['880422015', 2019-03-28T17:07, 'None', '']
    Browser Card Info Bar ['2140680811', 2020-02-19T15:19, 'None', '']
    Browser MaximizeHide TableEditor ['1819291495', 2019-12-22T12:15, 'None', '']
    Browser Resizer ['1435775540', 2020-02-08T01:39, 'None', '']
    Browser Search Box Quick Insert Tag Deck Notetype ['1052724801', 2020-04-08T18:25, 'None', '']
    CTRLF5 to Refresh the Browser ['1347728560', 2019-08-08T20:58, 'None', '']
    Card Info During Review ['2179254157', 2019-12-17T09:31, 'None', '']
    Change Card Creation Times ['217650262', 2019-11-17T08:01, 'None', '']
    Change Order of Review Cards in Regular Decks ['3731265543', 2019-11-25T23:30, 'None', '']
    Change cards decks prefix ['1262882834', 2019-08-10T10:14, 'None', '']
    Choose in which order you see new cards ['1665261045', 2020-02-16T04:40, 'None', '']
    Clean the default deck ['683170394', 2019-12-07T17:03, 'None', '']
    Clickable Tags v11 ['380714095', 2020-03-22T14:56, 'None', '']
    Color Confirmation ['1084228676', 2020-03-14T13:13, 'None', mod]
    Colorful toolbars ['388296573', 2019-11-21T13:23, 'None', '']
    Content Injector ['166455199', 2020-01-13T10:23, 'None', '']
    Control Audio Playback Extended ['1413404470', 2020-02-12T12:57, 'None', '']
    Convert Subdecks to Tag Hierarchy ['1172858842', 2020-02-23T23:03, 'None', '']
    Cookie monster ['1501583548', 2019-08-10T10:00, 'None', '']
    Create Filtered Deck from the Browser ['861263371', 2019-08-10T10:15, 'None', '']
    Cross Profile Search and Import ['310394744', 2019-08-10T10:00, 'None', '']
    Custom Background Image and Gear Icon ['1210908941', 2020-03-21T01:58, 'None', '']
    Customize Editor Stylesheet ['1215991469', 2019-11-23T11:38, 'None', '']
    Customize Sidebar ['1988760596', 2020-01-15T22:01, 'None', mod]
    Directly review without going through overview page ['1024346707', 2019-11-07T09:36, 'None', '']
    DrawingPad ['370778248', 2020-01-31T15:35, 'None', '']
    Echelon - Improved Hierarchical Tags for Anki 21 - Unsupported ['829533913', 2019-11-18T00:54, 'None', mod]
    Edit audios in the external editor ['1502086928', 2020-02-12T12:57, 'None', '']
    Editor Autocomplete ['924298715', 2020-02-16T10:55, 'None', '']
    Editor Autocomplete for Anki 21 ['925419294', 2019-12-04T15:58, 'None', '']
    Enhanced cloze 21 ['2062736101', 2019-08-14T13:53, 'None', '']
    Export cards selected in the Browser ['1983204951', 2020-02-04T17:14, 'None', '']
    Export deck to HTML ['1897277426', 2020-03-09T03:00, 'None', '']
    Export deck to audio Anki 21 ['2095022381', 2020-03-09T02:55, 'None', '']
    Fast Word Query Multi-threaded queries for words from local or web dictionaries ['1807206748', 2019-08-07T22:16, 'None', '']
    Feed to Anki for Word of the Day RSS ['1895937979', 2019-08-10T10:36, 'None', mod]
    Field History ['1247884413', 2019-08-15T11:03, 'None', '']
    Fill the blanks - Multiple typecloze support ['1933645497', 2020-03-14T02:20, 'None', '']
    Find Missing Words ['754868802', 2019-11-18T07:34, 'None', '']
    Find words and create deck ['1572743214', 2020-02-12T12:57, 'None', '']
    Five Finger Silver New Answer Button Graph ReLrn Stats ReMemorize Plan9-0 ['2047355215', 2019-08-19T14:28, 'None', '']
    Fluent Forever - Open Websites ['143682703', 2019-12-22T18:22, 'None', '']
    Full Screen Toggle ['1612375712', 2019-08-07T23:20, 'None', '']
    GODMODE faster shortcuts and Cloze switching ['1508677152', 2020-01-29T16:25, 'None', mod]
    HTML Editor Tweaks v02 ['410936778', 2020-03-22T14:58, 'None', '']
    Hide UI ['406504899', 2020-02-23T07:28, 'None', '']
    Hierarchical Tags 2 ['594329229', 2020-04-11T18:36, 'None', '']
    Image Style Editor ['1593969147', 2020-03-06T05:51, 'None', '']
    Import Video Add new cards from a video with subtitles ['726150594', 2019-08-15T09:24, 'None', '']
    Incremental Reading v4103 ['935264945', 2019-08-10T18:13, 'None', '']
    Insert Randomized Lists multiple choice ['1280092568', 2019-08-10T10:00, 'None', '']
    Interactive Cards for language learning ['1819109876', 2019-08-07T23:35, 'None', '']
    Keep model of add cards ['424778276', 2019-08-12T10:24, 'None', '']
    Knockout Deck Subset Learning ['899747926', 2019-08-09T09:44, 'None', '']
    Large and Colorful Buttons ['1829090218', 2019-08-10T20:52, 'None', '']
    Le Petit Debugger ['2099968349', 2019-08-10T09:47, 'None', '']
    Life Drain ['715575551', 2019-10-10T14:53, 'None', '']
    MIA Dictionary ['1655992655', 2020-04-06T23:48, 'None', '']
    MIA Retirement ['1666520655', 2020-03-11T00:26, 'None', '']
    Maximum image height and width in card editor ['229181581', 2020-02-20T09:10, 'None', '']
    Minimize to tray ['85158043', 2020-02-15T16:28, 'None', '']
    More Answer Buttons with colors ['1550991760', 2019-10-28T10:24, 'None', '']
    More Overview Stats 21 ['738807903', 2019-08-14T07:50, 'None', '']
    More consistent cards generation ['1713990897', 2019-08-12T17:22, 'None', '']
    MorphMan for Anki 21 ['900801631', 2020-02-24T20:07, 'None', '']
    Newline in strings in add-ons configurations ['112201952', 2020-03-19T19:41, 'None', '']
    Night Mode ['z1496166067', 2020-01-29T22:27, 'None', '']
    Nonconsecutive Search in Study Deck ['984507853', 2020-01-04T20:09, 'None', '']
    Number of Decks ['1358364553', 2019-12-11T20:01, 'None', '']
    Numeric Keypad Remapping  ease bugs fixed ['653807651', 2019-08-16T12:10, 'None', mod]
    Open Added Today from Reviewer ['861864770', 2019-12-07T17:03, 'None', '']
    Open linked pdf docx epub etc externally old version with config via json ['1994996371', 2019-09-15T12:38, 'None', '']
    Oxford Dictionaries ['1515522458', 2020-01-03T15:30, 'None', '']
    Pennywise Add Only Once Option ['1032766035', 2019-08-13T10:24, 'None', '']
    Pokemanki ['633922407', 2020-03-04T04:10, 'None', '']
    Pomodore Tomato Clock Port ['811976365', 2019-08-25T19:07, 'None', '']
    Pop-up Dictionary beta ['153625306', 2019-08-09T18:26, 'None', mod]
    Pop-up Wikipedia Beta ['395343016', 2020-01-14T00:26, 'None', '']
    Postpone cards review ['1152543397', 2020-04-05T01:11, 'None', '']
    Pretzel Logic Retention Benchmarking ['1168065472', 2020-04-02T00:43, 'None', '']
    Progress Bar ['2091361802', 2017-08-29T21:00, 'None', '']
    Quantified Self add-on - export your review log fork for Anki 21 ['1806723271', 2020-02-29T14:23, 'None', '']
    Quick  Easy ['1834460432', 2019-08-07T23:01, 'None', mod]
    Quick note and deck buttons Fork for 21 ['1331140127', 2020-02-22T12:32, 'None', mod]
    Quick tagging 21 ['304770511', 2020-02-11T07:17, 'None', mod]
    Rebuild All  Empty All for Anki 21 ['1810938259', 2019-11-14T07:56, 'None', '']
    Redesign ['1914733489', 2020-02-29T23:46, 'None', '']
    Reflect On Mee - Helps Alleviate RGS ['86140736', 2019-08-07T23:26, 'None', '']
    Refocus Card when Reviewing 21 ['1642550423', 2019-08-07T23:20, 'None', '']
    Remaining time for Anki 21 ['1508357010', 2020-03-28T13:43, 'None', '']
    Replay buttons on card ['498789867', 2019-08-07T22:10, 'None', '']
    Reset Ease ['947935257', 2019-11-17T08:47, 'None', '']
    Review Heatmap ['review_heatmap', 2018-10-28T23:06, 'None', '']
    Review Hotmouse ['1928346827', 2020-01-20T14:26, 'None', '']
    Reviewer Context Menu Search ['359618071', 2019-08-19T14:31, 'None', '']
    Right Hand Answer Shortcuts ['1816717670', 2020-03-14T19:45, 'None', '']
    Search in formatted text in any fields ['1126516755', 2020-03-10T22:36, 'None', '']
    Sentence ATM ['1209404305', 2019-11-06T16:16, 'None', '']
    Separate Learn and Relearn in the Answer Buttons graph 21 ['403491806', 2019-11-10T08:55, 'None', '']
    Set Randomizer v22 ['1164532380', 2019-12-01T21:33, 'None', mod]
    Small Potatoes Show learning step count in days ['75718778', 2020-04-02T20:53, 'None', '']
    Sort by Card Type Note Type or Deck Name ['1872117511', 2019-12-02T18:21, 'None', '']
    Speed Focus Mode auto-alert auto-reveal auto-fail ['1046608507', 2019-08-18T09:28, 'None', '']
    Spell Checker ['spell_checker', 0, 'None', '']
    Spelling Police ['1410276506', 2020-03-03T03:13, 'None', '']
    Stats Overview Pie Graph with Distinct Learning and-or Relearning Sections ['1828603731', 2019-08-14T07:50, 'None', '']
    Straight Reward Ease Factor grows at 5 straight Good rating in review ['1130426680', 2019-08-07T22:57, 'None', '']
    Syllabus - Anki Statistics and Export by Tag and Deck ['668156440', 2020-01-12T12:47, 'None', mod]
    Symbols As You Type ['2040501954', 2020-03-18T04:36, 'None', '']
    Symbols ['1461102936', 2019-08-11T21:28, 'None', mod]
    TagSelectorV2 ['1022577188', 2019-08-07T22:57, 'None', '']
    The KING of Button Add-ons ['374005964', 2020-03-20T15:34, 'None', '']
    Throughput Historical ['485345973', 2019-08-18T09:28, 'None', '']
    Throughput Monitor ['826650564', 2019-08-22T07:17, 'None', '']
    Touch Screen - draw and write on your cards ['1631622775', 2020-01-05T11:21, 'None', '']
    Trigger and actions change one card depending on what occurs on another card ['1981494159', 2020-03-20T01:19, 'None', '']
    Undelete deleted note ['912930620', 2019-08-12T13:35, 'None', '']
    Update add-ons when collection is synchronized ['1847544206', 2019-12-07T17:03, 'None', '']
    Update lookup Tags ['1715601279', 2019-12-11T08:29, 'None', '']
    Web Browser - Search terms Import texts and images automatically ['864545277', 2020-01-18T19:52, 'None', mod]
    Web Query  ['627484806', 2019-08-11T15:00, 'None', '']
    Word search and wildcard field name 21 ['114749949', 2020-01-28T21:40, 'None', '']
    Ze Add note id ['886709118', 2019-11-15T07:30, 'None', '']
    Zoom 21 ['1846592880', 2020-01-13T10:16, 'None', mod]
    adjust some colors in the stats window ['1202569752', 2019-11-25T12:28, 'None', '']
    ambossankiaddon file ['1845101054', 2019-10-27T12:29, 'None', '']
    auto rate typed answer ['1836029849', 2019-09-04T09:04, 'None', '']
    batch change fonts ['1479177943', 2019-11-23T11:36, 'None', '']
    bury future new sibs of learned ['bury future new siblings of learned', 2019-11-25T09:52, 'None', '']
    bury future new sibs of learning ['bury future new siblings of learning and day learning', 2019-11-25T09:52, 'None', '']
    deck and card info sidebar during review ['673114053', 2020-03-22T13:12, 'None', '']
    ease modify extended ['1124029139', 2019-11-21T13:21, 'None', '']
    export all notetypesmodels and deckoptionsgroups without notes or media ['870620435', 2019-11-23T11:34, 'None', '']
    export cardsnotes from browser with metadata to csv or xlsx ['1967530655', 2019-11-25T09:39, 'None', '']
    movie2anki ['1846227036', 2019-08-15T10:11, 'None', '']
    put ALL due learning cards first modified v2 support ['1807121582', 2019-11-25T09:35, 'None', '']
    show new siblings in order  no same day spacingrandomization for new siblings ['268644742', 2020-02-18T19:41, 'None', '']
    zzz Make It Harder To Quit ['1381659795', 2020-02-23T07:28, 'None', '']

```

**Comment by hssm** [comment]
It doesn't look like it's coming from the Advanced Browser add-on (but since it overrides the default browser it appears as the top-level object where it failed).

If you remove it, do you still have the same error?

**Comment by galantra** [comment]
You are right. And thanks for making me notice that the last-mentioned add-on is not always involved in the interference-.
